ami_configs/bld-linux64.json

```diff
@@ -1,30 +1,6 @@
 {
-    "us-west-1": {
-        "ami": "ami-fe393ebb",
-        "instance_type": "c3.xlarge",
-        "arch": "x86_64",
-        "distro": "centos",
-        "root_device_type": "instance-store",
```

Comment:
So, should I not delete this clause?  Create a base AMI in usw1 and update the AMI id even though it's not used?